CHANGES

```diff
@@ -42,6 +42,8 @@ Upcoming release: BINARY COMPATIBLE
 * Removed obsolete define `HAVE_AUTOCONF`
 * Removed user address space reserved slots restriction on 40bit PA platforms when KernelArmHypervisorSupport is set.
   This change is reflected in the definition of the seL4_UserTop constant that holds the largest user virtual address.
+* aarch32 VM fault messages now deliver original (untranslated) faulting IP address in a hypervisor context, matching
+  aarch64 behaviour.
```

Comment:
Potentially: 

```suggestion
* aarch32 VM fault messages now deliver original (untranslated) faulting IP in a hypervisor context, matching
  aarch64 behaviour.
```

(it sounded like a network IP address to me on first read, but it's the instruction pointer, of course)

Reply:
Good point (ha). Will update after tests, thanks!